bin/srwr_process.php

```diff
@@ -1,10 +1,10 @@
 <?php
 
-require_once __DIR__ . '/../../conf/config';
-require_once __DIR__ . '/../fns.php';
-require __DIR__ . '/../../vendor/autoload.php';
+require_once __DIR__ . '/../conf/config';
+require_once __DIR__ . '/../web/fns.php';
+require __DIR__ . '/../vendor/autoload.php';
 
-$path = './srwr_data/';
+$path = '../web/streetmanager/srwr_data/';
```

Comment:
Is there a reason this path has to be in web? It's not ever publicly accessed, is it?

Reply:
So would `../../data` be the location?

Reply:
Sounds okay to me, yeah, will also stay there during deploys (a timestamped deploy creates a whole new clone of the repository each time)